src/activities/quizzes/ipRestrictions/QuizIpRestrictionsEntity.js

```diff
@@ -35,10 +35,10 @@ export class QuizIpRestrictionsEntity extends Entity {
 
 		return ipRestrictionEntities.map(({ properties }, index) => {
 
-			const { start = '', end = '' } = properties;
+			const { start, end } = properties;
 			return {
-				start,
-				end,
+				start: start || '',
```

Comment:
Changed this fall back to here as the default values before only checked for `undefined` which means an IP is sometimes `null`. This way should guarantee any null or undefined ranges will be treated as an empty string.